src/main/java/org/apache/sysml/api/mlcontext/MLContext.java

```diff
@@ -334,7 +334,7 @@ public MLResults execute(Script script, ScriptExecutor scriptExecutor) {
 		try {
 			executionScript = script;
 
-			Long time = new Long((new Date()).getTime());
+			Long time = new Long(System.currentTimeMillis());
```

Comment:
I suggest, better to move `time` variable to `if` block, because it uses only for setting script name